### PR TITLE
Add link to GithubPages starter repo

### PR DESCRIPTION
**Pull request by spacegiant**
Created new section in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +23,8 @@ Templates, random tables, character sheets and other stuff for playing solo RPGs
 - Homepage.
 - Timelines.
 - Tag Wrangler.
-- Gallery.
\ No newline at end of file
+- Gallery.
+
+## Extras
+
+- [Publish to Github Pages](https://github.com/spacegiant/obsidian-github-pages-starter)
```
Comment (gbraad) [inline README.md:30]: Why do you link to this?

Comment (spacegiant) [reply README.md:30]: This was requested addition on discord. This is instructions on how to automate publishing your obsidian actual play notes to a GitHub pages website.

Comment (spacegiant) [reply README.md:30]: Feel free to add the page directly if that makes more sense

Comment (gbraad) [reply README.md:30]: It was unclear to an outsider, as there are no instructions to explain this.

**Pull request merged**